### PR TITLE
[RFR][NOTEST] Update+Fix template_upload_vsphere

### PR DESCRIPTION
**Pull request by mshriver**
Depends on wrapanapi update: ~~https://github.com/ManageIQ/wrapanapi/pull/153~~  (in version 2.1.5)

Uses `add_disk_to_vm()` method provided by wrapanapi update.

Removes unused methods that were using a removed wrapanapi virtualcenter.api property.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 3069c2d878af1a0fb8f169c26c8827fed5e30f69:

`scripts/template_upload_vsphere.py`:
- :red_circle: Line 117:5: **F841** *local variable 'task' is assigned to but never used*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: scripts/template_upload_vsphere.py [hunk 1]
```diff
@@ -227,24 +130,6 @@ def check_kwargs(**kwargs):
     return True
 
 
-def update_params_api(client, **kwargs):
```
Comment (mshriver) [inline scripts/template_upload_vsphere.py]: Unused

File: scripts/template_upload_vsphere.py [hunk 2]
```diff
@@ -112,110 +107,18 @@ def upload_ova(hostname, username, password, name, datastore,
...
+    if rc:
+        print('VSPHERE:{} Added disk to vm {}'.format(provider, name))
     else:
-        client.api.logout()
-        print(" VSPHERE:{} Failed to add disk".format(provider))
-        return False
+        print(" VSPHERE:{} Failure adding disk, result: {}".format(provider, result))
 
-
-def make_template(client, name, provider):
```
Comment (mshriver) [inline scripts/template_upload_vsphere.py]: All unused after removing unused update_params_api

**Comment by mshriver** [comment]
Merging as-is for testing upload on new sprout providers for CFME QE

**Pull request merged**